lib/i18n/backend/base.rb

```diff
@@ -173,11 +173,12 @@ def resolve(locale, object, subject, options = EMPTY_HASH)
 
         # Picks a translation from a pluralized mnemonic subkey according to English
         # pluralization rules :
-        # - It will pick the :one subkey if count is equal to 1.
+        # - It will pick the :zero subkey where count is equal to 0 and there
+        #   is a :zero subkey present. This behaviour is not standard with
+        #   regards to the CLDR pluralization rules.
+        # - It will pick the :one subkey if count is equal to 1 and there is a
+        #   :one subkey present.
         # - It will pick the :other subkey otherwise.
-        # - It will pick the :zero subkey in the special case where count is
-        #   equal to 0 and there is a :zero subkey present. This behaviour is
```

Comment:
I'm not sure whether this "not standard" comment means the behaviour is not standard in relation to the CLDR English pluralization rules (because there is a `zero` subkey) or if there is some sort "generic/unknown locale" rule in CLDR that it's deviating from in some way. If I could get an answer on that I could update the comment to be clearer.